@here/harp-olp-utils/lib/OlpCopyrightProvider.ts

```diff
@@ -6,6 +6,7 @@
 
 import { AreaCopyrightInfo, CopyrightCoverageProvider } from "@here/harp-mapview";
 import { CatalogClient, DataStoreContext, HRN } from "@here/olp-sdk-dataservice-read";
+import { EnvironmentName } from "@here/olp-sdk-dataservice-read/dist/@here/olp-sdk-dataservice-read/lib/getEnvLookupUrl";
```

Comment:
Is this a temporary hack and this will be in the public API of olp-sdk-dataservice-read at some point?

Reply:
New version of `olp-sdk-dataservice-read` has a better structure so this can be imported from it's `index.ts`. However, new version of `olp-sdk-dataservice-read` has significantly changed API so the whole `OlpCopyrightProvider` and `OlpDataProvider` should be rewritten to adapt to it. This is a quick fix which uses version specified in `package.json` and doesn't require major refactoring.

Reply:
We've found solution offline:
```
import { ..., DataStoreContextParams, ... } from "@here/olp-sdk-dataservice-read";

export interface OlpCopyrightProviderParams {
    ...
   environment?: DataStoreContextParams["environment"];
}
```

that is both typesafe and uses published interfaces only and most importantly doesn't depend on arbitrary physical layout of 3rdparty package.